backend_new/src/services/application-csv-export.service.ts

```diff
@@ -0,0 +1,653 @@
+      {
+        path: 'alternateContact.otherType',
+        label: 'Alternate Contact Other Type',
+      },
+      {
+        path: 'alternateContact.emailAddress',
+        label: 'Alternate Contact Email Address',
+      },
+      {
+        path: 'alternateContact.phoneNumber',
+        label: 'Alternate Contact Phone Number',
+      },
+      {
+        path: 'alternateContact.address.street',
+        label: `Alternate Contact Street`,
+      },
+      {
+        path: 'alternateContact.address.street2',
+        label: `Alternate Contact Street 2`,
+      },
+      {
+        path: 'alternateContact.address.city',
+        label: `Alternate Contact City`,
+      },
+      {
+        path: 'alternateContact.address.state',
+        label: `Alternate Contact State`,
+      },
+      {
+        path: 'alternateContact.address.zipCode',
+        label: `Alternate Contact Zip Code`,
+      },
+      {
+        path: 'income',
+        label: 'Income',
+      },
+      {
+        path: 'incomePeriod',
+        label: 'Income Period',
+        format: (val: string): string =>
+          val === 'perMonth' ? 'per month' : 'per year',
+      },
+      {
+        path: 'accessibilityMobility',
+        label: 'Accessibility Mobility',
+      },
+      {
+        path: 'accessibilityVision',
+        label: 'Accessibility Vision',
+      },
+      {
+        path: 'accessibilityHearing',
+        label: 'Accessibility Hearing',
+      },
+      {
+        path: 'householdExpectingChanges',
+        label: 'Expecting Household Changes',
+      },
+      {
+        path: 'householdStudent',
+        label: 'Household Includes Student or Member Nearing 18',
+      },
+      {
+        path: 'incomeVouchers',
+        label: 'Vouchers or Subsidies',
+      },
+      {
+        path: 'preferredUnitTypes',
+        label: 'Requested Unit Types',
+        format: (val: UnitType[]): string => {
+          return val
+            .map((unit) => this.unitTypeToReadable(unit.name))
+            .join(',');
+        },
+      },
+    ];
+
+    // add preferences to csv headers
+    multiSelectQuestions
+      .filter((question) => question.applicationSection === 'preferences')
+      .forEach((question) => {
+        headers.push({
+          path: `preferences.${question.id}.claimed`,
+          label: `Preference ${question.text}`,
+          format: (val: boolean): string => (val ? 'claimed' : ''),
+        });
+        /**
+         * there are other input types for extra data besides address
+         * that are not used on the old backend, but could be added here
+         */
+        question.options
+          ?.filter((option) => option.collectAddress)
+          .forEach(() => {
+            headers.push({
+              path: `preferences.${question.id}.address`,
+              label: `Preference ${question.text} - Address`,
+              format: (val: ApplicationMultiselectQuestion): string =>
+                this.multiselectQuestionFormat(val),
```

Comment:
For this and the other preference fields it is coming back as `undefined undefined` 
<img width="531" alt="image" src="https://github.com/bloom-housing/bloom/assets/42942267/f91d99ce-1ea3-407b-a418-ed745ef7278f">